spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/OrderedEnvironmentRepository.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.config.server.environment;
+
+import org.springframework.core.Ordered;
+
+/**
+ * A prioritized {@link EnvironmentRepository}.
+ * @author Ryan Baxter
+ */
+public interface OrderedEnvironmentRepository extends EnvironmentRepository, Ordered {
```

Comment:
I think this interface isn't needed is it? I think it could be removed and all your tests would still pass.